### PR TITLE
Split ZKSYNC_RPC_ADDRESS into mainnet and testnet

### PR DESCRIPTION
**Pull request by maaktweluit**
To allow a custom RPC for each network

### CODE REVIEW

File: core/payment-driver/zksync/src/zksync/wallet.rs [hunk 1]
```diff
@@ -222,13 +222,20 @@ pub async fn verify_tx(tx_hash: &str, network: Network) -> Result<PaymentDetails
...
 fn get_provider(network: Network) -> Provider {
-    let provider: Provider = match env::var("ZKSYNC_RPC_ADDRESS").ok() {
+    let provider: Provider = match get_rpc_from_env(network) {
         Some(rpc_addr) => Provider::from_addr(rpc_addr),
         None => Provider::new(get_zk_network(network)),
     };
     provider.clone()
 }
 
+fn get_rpc_from_env(network: Network) -> Option<String> {
```
Comment (pnowosie) [inline core/payment-driver/zksync/src/zksync/wallet.rs:232]: Little bit more descriptive name
```suggestion
fn get_rpc_address_from_env(network: Network) -> Option<String> {
```

**Review by pnowosie — APPROVED** [review]
Only rename suggested. Good thing!

File: core/payment-driver/zksync/src/zksync/wallet.rs (hunk 1 above)
Comment (maaktweluit) [reply core/payment-driver/zksync/src/zksync/wallet.rs]: good one! updated it to `addr` to not make the name too long ( and be consistent with zksync's `get_rpc_addr` )

**Review by Wiezzel — COMMENTED** [review]
:ok_hand:

**Pull request merged**